src/govuk/components/warning-text/warning-text.yaml

```diff
@@ -34,6 +34,8 @@ examples:
   - name: attributes
```

Comment:
I think the 'text only' example in this file needs an `iconFallbackText` or possibly should just be removed, along with the test?

Reply:
Addressed in https://github.com/alphagov/govuk-frontend/pull/1990/commits/da7b4f6369886dfe4f8e31e6095789ef88e0238a